ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -409,7 +414,10 @@ export default class ConfirmTransactionBase extends Component {
       );
     };
 
-    const renderTotalMaxAmount = (useMaxFee) => {
+    const renderTotalMaxAmount = (
+      useMaxFee,
+      isBoldTextAndNotOverridden = false,
```

Comment:
Minor, but in cases like these we have many positional boolean arguments, should we move to an object so it's more explicit when calling what the arguments mean?

Reply:
updated [c27d9a1](https://github.com/MetaMask/metamask-extension/pull/24241/commits/c27d9a166e9719bfd4a32e089eece038b153d9c8)